lxd/cluster/connect.go

```diff
@@ -143,7 +143,7 @@ func SetupTrust(cert, targetAddress, targetCert, targetPassword string) error {
 
 	block, _ := pem.Decode([]byte(cert))
 	if block == nil {
-		return errors.Wrap(err, "failed to decode certificate")
+		return errors.Errorf("failed to decode certificate")
```

Comment:
Can you use `fmt.Errorf` instead? That'd be more consistent with what we've done in the rest of the codebase.

Reply:
@stgraber done.